section3/reflection.md

```diff
@@ -1,17 +1,33 @@
 ## Section 3 Reflection
 
-1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
+1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique? <br>
+Felt a little bit different, I didn't do much during the breaks.  Also got distracted easily
 
-2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
+2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)? <br>
+When the breaks came, I found myself doing the things I usually do when I get distracted (look at twitter, check texts, etc.)
```

Comment:
Ooh interesting. Are you ok with doing those things on breaks, or are the breaks too short/then you have a hard time getting focused back in?

A couple years ago I read some book about productivity (can't even tell you the name of it) but it inspired me to not check non-essentials (social media, personal email) until the end of the day. It talked about how once you check it first thing in the am or on lunch break, your brain will want to keep coming back, etc. So changing that up helped me stay focused during the day a lot. Probably not for everyone but 🤷‍♀️